game/squadrons/airwing.py

```diff
@@ -15,13 +16,26 @@
 
 
 class AirWing:
+
+    squadron_def_generator: SquadronDefGenerator
+
     def __init__(self, player: bool) -> None:
+        super().__init__()
```

Comment:
no-op

Reply:
Was a left over from the previous observer inherit. removed it :)